app/views/funding/services/show.html.haml

```diff
@@ -18,14 +18,17 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
-.panel.panel-default
-  .panel-heading.justify-content-between
-    %h4.panel-title
-      = t(:funding)[:download][:header]
-    = link_to "/funding?back=true", title: "Back to funding opportunities" do
-      %span.glyphicon.glyphicon-arrow-left{aria: {hidden: "true"}}
-      = t(:actions)[:back]
-  .panel-body
+
+%header.d-flex.mb-2.default
+  %h2.p-2.mr-auto
+    = t('funding.download.header')
+  %h4.p-2
+    = link_to "/funding?back=true" do
+      = icon('fas', 'arrow-left mr-2')
+      = t('funding.download.back')
+
+.card
+  .card-header.bg-default.text-dark.h4
     = render 'funding/services/title', service: @service
+  .card-body.mt-3
```

Comment:
This card/table I'm not entirely sure about. Can you try something and let me know how it looks? To make the table consistent like on the other card, can you move it outside the `.card-body` so that it's directly under the `.card`? Then can you wrap the two buttons inside a `.card-body` so that they have the padding we want? That will probably put a line between the buttons and the table, but I'm wondering if you add a `.border-bottom-0` to the `.card-body` if that would look okay?

Reply:
I moved the table outside the .card-body and placed it directly under the .card, It looks okay
![image](https://user-images.githubusercontent.com/7648699/65704000-5da87180-e04b-11e9-9465-92e2e9fd1c17.png)

Reply:
@ying-pbrc Are you sure the table isn't still inside of the `.card-body`? It looks like it is from that screenshot because of the padding around the sides of the table

Reply:
updated screenshot, just pushed the new commit to upstream
![image](https://user-images.githubusercontent.com/7648699/65797989-2bc00980-e136-11e9-83e1-eef2db22f6c5.png)